### PR TITLE
Add gRPC context propagation tests for streaming APIs

### PR DESCRIPTION
**Pull request by misterpoe**

### CODE REVIEW

**Comment by misterpoe** [comment]
@matthewloring @ofrobots @cristiancavalli PTAL

File: test/standalone/test-grpc-context.js [hunk 1]
```diff
@@ -18,61 +18,158 @@
...
   grpc015: require('../hooks/fixtures/grpc0.15')
 };
 
+var grpcPort = 50051;
+var protoFile = __dirname + '/../fixtures/test-grpc.proto';
+var debugCount = 0;
+var client, grpcServer, server;
+
+agent.logger.debug = function(error) {
+  if (error.indexOf('http') !== -1) {
```
Comment (matthewloring) [inline test/standalone/test-grpc-context.js]: Could we strengthen this check to ensure the we are only failing to log http requests made to the express app. (Ensure that no requests are associated with the url www.google.com).

**Comment by matthewloring** [comment]
LGTM.

File: test/standalone/test-grpc-context.js (hunk 1 above)
Comment (misterpoe) [inline test/standalone/test-grpc-context.js]: Done

**Pull request merged**